processors/filterprocessor/config.go

```diff
@@ -0,0 +1,18 @@
+package filterprocessor
+
+import "github.com/algorand/indexer/processors/filterprocessor/expression"
+
+// SubConfig is the configuration needed for each additional filter
+type SubConfig struct {
+	// The tag of the struct to analyze
+	FilterTag string `yaml:"tag"`
+	// The type of expression to search for "const" or "regex"
```

Comment:
Looks like `const` this was renamed. Also, since you have comments anyway, could you add the variable names so that they matches the godoc format?